### PR TITLE
Added another missing __init__.py

### PR DESCRIPTION
**Pull request by micheles**
So the tests inside `mgpe` were not run :-(
And of course they were broken.
Closes https://github.com/gem/oq-engine/issues/4394 too.

### CODE REVIEW

**Comment by klunk386** [comment]
LGTM

File: openquake/hazardlib/tests/gsim/mgmpe/generic_gmpe_avgsa_test.py [hunk 1]
```diff
@@ -14,16 +14,19 @@
...
 
+data = pathlib.Path(__file__).parent / 'data'
+
 
 class GenericGmpeAvgSATestCase(unittest.TestCase):
 
     def test_calculation_Akkar(self, avg_periods="0.05,0.15,1.0,2.0,4.0"):
 
-        DATA_FILE = 'Data/GENERIC_GMPE_AVGSA_MEAN_STD_TOTAL_AKKAR.csv'
+        DATA_FILE = data/'GENERIC_GMPE_AVGSA_MEAN_STD_TOTAL_AKKAR.csv'
```
Comment (daniviga) [inline openquake/hazardlib/tests/gsim/mgmpe/generic_gmpe_avgsa_test.py]: That's wrong

File: openquake/hazardlib/tests/gsim/mgmpe/generic_gmpe_avgsa_test.py [hunk 2]
```diff
@@ -71,7 +74,7 @@ def test_calculation_Akkar_valueerror(self):
 
     def test_calculation_Baker_Jayaram(self):
 
-        DATA_FILE = 'Data/GENERIC_GMPE_AVGSA_MEAN_STD_TOTAL_BAKER_JAYARAM.csv'
+        DATA_FILE = data/'GENERIC_GMPE_AVGSA_MEAN_STD_TOTAL_BAKER_JAYARAM.csv'
```
Comment (daniviga) [inline openquake/hazardlib/tests/gsim/mgmpe/generic_gmpe_avgsa_test.py]: Same as above

File: openquake/hazardlib/tests/gsim/mgmpe/generic_gmpe_avgsa_test.py (hunk 1 above)
Comment (micheles) [reply openquake/hazardlib/tests/gsim/mgmpe/generic_gmpe_avgsa_test.py]: No, it is the magic of pathlib ;-)

File: openquake/hazardlib/tests/gsim/mgmpe/generic_gmpe_avgsa_test.py (hunk 2 above)
Comment (micheles) [reply openquake/hazardlib/tests/gsim/mgmpe/generic_gmpe_avgsa_test.py]: Same as above ;-)

**Pull request merged**